packages/components/src/local/molecules/channel-coa-message-detail/helpers/messages_helper.ts

```diff
@@ -0,0 +1,106 @@
+import { COUNTER_MESSAGE } from '@serge/config'
```

Comment:
Hello @lilitkarapetyan 

Once the game managers have duplicated their messages, I expect to throw "Duplicate" button away.

But, the code in this helper function looks like should keep it.  Can you explain what it is doing please?

Reply:
It's special type of message witch one contain counter for COA messages, it's for creating queue when players updating coa message in some time do to not have conflicts and keep righ counter
https://github.com/serge-web/serge/blob/develop/packages/client/src/api/messages_helper.ts